controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -388,6 +388,21 @@ def test_get_object_by_id_returns_none(self):
     def test_property(self):
         self.assertEqual(XIVArrayMediator.port, 7778)
 
+    def test_get_host_by_name_success(self):
+        host = utils.get_mock_xiv_host("test_host_1", "iqn.test.1", "")
+        self.mediator.client.cmd.host_list.return_value = Mock(as_single_element=host)
+        host = self.mediator.get_host_by_name('test_host_1')
+        self.assertEqual(host.name, "test_host_1")
+        self.assertEqual(host.connectivity_types, ['iscsi'])
+        self.assertFalse(host.initiators.is_array_nvme_nqn_match([]))
```

Comment:
could this ever be `True` when called with an empty list?

Reply:
changed

Reply:
I'm not sure we want the members to be public

Reply:
also the tests [should mock the host](https://github.com/IBM/ibm-block-csi-driver/pull/495#discussion_r859810138) anyway

Reply:
CSI-4250